packages/clerk-js/src/core/clerk.ts

```diff
@@ -913,9 +913,10 @@ export default class Clerk implements ClerkInterface {
     if (this.session) {
       const lastId = this.session.id;
       this.session = newClient.activeSessions.find(x => x.id === lastId);
-      this.organization = (this.session?.user.organizationMemberships || [])
-        .map(om => om.organization)
-        .find(org => org.id === this.session?.lastActiveOrganizationId);
+      this.organization =
+        (this.session?.user.organizationMemberships || [])
+          .map(om => om.organization)
+          .find(org => org.id === this.session?.lastActiveOrganizationId) || null;
```

Comment:
Excellent catch. AFAIK we use the same snippet one more time in clerk.ts, can you update that as well?
This duplication is slightly problematic in this case, but I'm currently working on improving this flow

Reply:
Think I found two more. Take a look.

Reply:
Are we good on this, can I merge?